TROUBLESHOOTING.md

```diff
@@ -0,0 +1,192 @@
+# Troubleshooting Common Issues
+
+## Using this Document
+
+Search for the error message you're getting and see if there's a match, or simply skim the table of contents below for topics that seem relevant to the issue you're having. Each issue section has steps to work around or fix the particular issue, and have examples of common error messages.
+
+If you do not find the issue below, try searching the issue tracker itself for potential duplicates before opening a new issue.
+
+If you're reading this document because you noticed an issue with npm's web site, you may want to let the [web team](https://github.com/npm/www/issues) know.
+
+### Updating this Document
+
+If you think something should be added here, make a PR that includes the following:
+
+0. a summary
```

Comment:
For consistency of editing this document like other documents in the repo:

Use the `1. 2. 3. 4` bullet points you did in the contributing guide, or switch the contributing guide to use `0. 0. 0. 0.`

For consistency of reading this document:

The starting word of each bullet should be capitalised like the rest :)